tests/test_hub.py

```diff
@@ -183,3 +185,13 @@ def test_get_allowed_tasks(tasks_dict, expected):
     tasks = get_allowed_tasks(tasks_dict)
 
     assert tasks == expected
+
+
+class ValidateYaml(TestCase):
+    readme_filepaths = []
+    for glob_path in ["measurements/*/README.md", "metrics/*/README.md", "comparisons/*/README.md"]:
+        readme_filepaths.extend(glob.glob(glob_path))
+    for readme_file in readme_filepaths:
+        with open(readme_file) as f_yaml:
+            x = yaml.safe_load_all(f_yaml)
+            assert type(next(x)) == dict
```

Comment:
could we make this a
```suggestion
            self.assertIsInstance(next(x), dict)
```